src/AzureDeprecation.Notifications/Functions/EventGridNotificationsFunction.cs

```diff
@@ -1,29 +1,35 @@
-using System;
-using System.Linq;
-using System.Net.Mime;
-using System.Threading.Tasks;
 using Arcus.EventGrid.Publishing;
 using AzureDeprecation.Contracts;
 using AzureDeprecation.Contracts.Messages.v1;
+using AzureDeprecation.Runtimes.AzureFunctions;
 using CloudNative.CloudEvents;
 using Microsoft.Azure.WebJobs;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
+using System.Net.Mime;
 
 namespace AzureDeprecation.Notifications.Functions
 {
-    public class EventGridNotificationsFunction
+    public partial class EventGridNotificationsFunction
```

Comment:
Why are we making it partial?

Reply:
For the `LoggerMessage`'s further down to work: https://learn.microsoft.com/en-us/aspnet/core/fundamentals/logging/loggermessage?view=aspnetcore-6.0